### DISCUSSION TITLE
Sustainability Strategy

### DISCUSSION THREAD

**Discussion by gasparnagy** [reqnroll/Reqnroll#8] [General]
Discussion, ideas, comments, feedback about the project sustainability strategy: https://go.reqnroll.net/sustainability

**Comment by clrudolphi** [comment]
I'd like to help. I have contributed a few minor bug fixes to SF and would like to continue working on this framework.

How will you approach the set of open Issues over in the SpecFlow project? Port them over to Reqnroll en masse, cherry-pick some, or not bother/start over?

**Reply by gasparnagy** [reply]
Good question. Porting all the issues here would just overwhelm the project, I think.

I would ask people to re-create "their" issues here (not necessarily the original creator of the issue, but someone who is affected) if they are still relevant here, so that we can try to address. 

We should be careful though to use the "Issues" only for issues, questions, ideas and feature requests should rather be captured here in the "Discussions" (in the ["Q & A"](https://github.com/orgs/reqnroll/discussions/categories/q-a) and the ["Ideas"](https://github.com/orgs/reqnroll/discussions/categories/ideas) sections). So that we can keep the number of open issues low.

**Comment by clrudolphi** [comment]
Further thoughts on generating movement from SpecFlow to Reqnroll:

- Would it be possible to get Tricentis to put up notices on the SpecFlow website and github pages to direct users over to Reqnroll?
- Suggest that press-release material get sent to the link blog authors, such as https://blog.cwa.me.uk/2024/02/09/the-morning-brew-3878/ and  https://www.alvinashcraft.com/2024/02/09/dew-drop-february-9-2024-4125/ to raise awareness within the SF community about the change.

**Reply by gasparnagy** [reply]
Thx for the hint with the blogs. I will contact them.

Tricentis: I haven't seen or received any comments on the fork announcement so far. But I agree, that if there will be a communication, we should ask for what you suggested.
